spec/refresher_spec.rb

```diff
@@ -5,70 +5,70 @@
   context "guess the collection" do
 
     it "Yes, it is a Object, but which class" do
-      expect([].is_a? _fill_in_object_).to be true
-      expect(Array.new.instance_of? _FILL_ME_IN_).to be true
+      expect([].is_a? Array).to be true
+      expect(Array.new.instance_of? Array).to be true
     end
 
     it "Yes, it is a Object, but which class" do
-      expect({}.is_a? _fill_in_object_).to be true
-      expect(Hash.new.instance_of? _FILL_ME_IN_).to be true
+      expect({}.is_a? Hash).to be true
+      expect(Hash.new.instance_of? Hash).to be true
     end
   end
 
   context "guess the type" do
     it "Yes, it is a Object, but which" do
-      expect(:class.is_a? _fill_in_object_).to be true
+      expect(:class.is_a? Object).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect("module".is_a? _fill_in_object_).to be true
+      expect("module".is_a? String).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect(1.is_a? _fill_in_object_).to be true
+      expect(1.is_a? Integer).to be true
     end
 
     it "Yes, it is a Object, but which" do
-      expect(1.5.is_a? _fill_in_object_).to be true
+      expect(1.5.is_a? Float).to be true
     end
   end
 
   context "the following are Array methods" do
     it "adds a thing to the end of the method" do
-      expect([]._fill_in_method_here_(1)).to eq [1]
+      expect([].push(1)).to eq [1]
     end
 
     it "removes an item from the end of an array" do
-      expect([1]._fill_in_method_here_).to eq 1
+      expect([1].pop).to eq 1
     end
 
     it "adds an item to the front of an array" do
-      expect([1]._fill_in_method_here_("banana")).to eq ["banana", 1]
+      expect([1].insert(0,"banana")).to eq ["banana", 1]
```

Comment:
Got it, insert is for a specific index/position. unshift is just for the
first element. Thanks

On Mon, Nov 16, 2015 at 12:59 PM, Michael Gee <notifications@github.com>
wrote:

> In spec/refresher_spec.rb
> <https://github.com/paircolumbus/RubyRefresher/pull/31#discussion_r44957850>
> :
>
> >      end
> >
> >      it "adds an item to the front of an array" do
> > -      expect([1]._fill_in_method_here_("banana")).to eq ["banana", 1]
> > +      expect([1].insert(0,"banana")).to eq ["banana", 1]
>
> I think unshift is a better choice, since that's all it does. I would
> suggest leaving insert for when you need to put the element elsewhere.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paircolumbus/RubyRefresher/pull/31/files#r44957850>.
>